receiver/awslambdareceiver/config.go

```diff
@@ -15,10 +15,10 @@ const (
 type Config struct {
 	// S3Encoding identifies the encoding of the S3 objects that trigger the Lambda.
 	//
-	// If S3Encoding is unspecified, the receiver will return an error for any S3 event notifications.
+	// If S3 data is in multiple formats (ex:- VPC flow logs, CloudTrail logs), you should deploy
+	// separate Lambda functions with specific extension configurations.
 	//
-	// If you have objects with multiple different encodings to handle, you should deploy
-	// separate Lambda functions with different configurations.
+	// If unspecified, the receiver falls back to work with CloudWatch Log subscription encoding extension.
```

Comment:
I know we went back and forth on this internally, but I'd really like to keep these things independent. Can we maintain the existing doc comment please? It's confusing to talk about CloudWatch logs in the same breath as S3 encoding - they're totally separate events.

Reply:
I am exploring the option described here - https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/44562#issuecomment-3587411621 . Let me know your view on this.